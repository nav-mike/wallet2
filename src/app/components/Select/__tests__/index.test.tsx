import * as React from 'react';
import { render } from '@testing-library/react';
import { Select } from '..';

describe('<Select />', () => {
  it('should be render and match the snapshot', () => {
    const select = render(<Select />);
    expect(select.container.firstChild).toMatchSnapshot();
  });
});
