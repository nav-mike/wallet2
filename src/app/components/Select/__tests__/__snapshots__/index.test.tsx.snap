// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Select /> should be render and match the snapshot 1`] = `
.c0 {
  border: 1px solid rgba(220,120,95,1);
  border-radius: 0.3rem;
  padding: 0.2rem;
}

.c0:hover {
  cursor: pointer;
}

<select
  class="c0"
/>
`;
